src/interactions/commands/player/play.ts

```diff
@@ -36,14 +36,25 @@ class PlayCommand extends BaseSlashCommandInteraction {
 
         await this.runValidators({ interaction, executionId }, [checkInVoiceChannel]);
 
-        let queue: GuildQueue = useQueue(interaction.guild!.id)!;
+        const player = useMainPlayer()!;
+        const queue: GuildQueue =
```

Comment:
What is the benefit of manual queue creation?

The way this is implemented with useQueue and useMainPlayer earlier has worked just fine, discord-player handles everything. If queue does not exist and player.play is called, a queue is created.